scripts/mono-symbolicate

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/local/bin/mono $MONO_OPTIONS /usr/local/lib/mono/4.5/mono-symbolicate.exe "$@"
```

Comment:
This should be a `.in` and use the macros to have the correct path when it's installed. See https://github.com/esdrubal/mono/blob/aotsp/scripts/mcs.in for example.